recipes/treesapp/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "treesapp" %}
+{% set version = "0.6.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b2cbce8aaab07e5b493f231b6af75724def82e5785f3b7f13f101c92d49d8550
+
+build:
+  number: 0
+  skip: True  # [not linux64]
+  entry_points:
+    - treesapp = treesapp.__main__:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python >=3.5
+  host:
+    - biopython >=1.68
+    - ete3 >=3.1.1
+    - numpy >=1.18.1
+    - pip
+    - pygtrie >=2.3
+    - python >=3.5
+    - samsum ==0.1.2
+    - scipy
+    - six >=1.14.0
+  run:
+    - python >=3.5
+    - {{ pin_compatible('numpy') }}
+    - scipy >=1.4.1
+    - biopython >=1.68
+    - pygtrie >=2.3.2
+    - ete3 >=3.1.1
+    - six >=1.14.0
+    - joblib >=0.14.1
+    - matplotlib-base >=3.1.2
+    - seaborn >=0.9.0
+    - hmmer >=3.1
+    - prodigal >=2.6.2
+    - fasttree >=2.1.9
+    - bwa >=0.7.3
+    - raxml ==8.2.12
+    - mafft >=7.407
+    - samsum ==0.1.2
+
+test:
+  imports:
+    - treesapp
+  commands:
+    - {{ name }} --help
+
+about:
+  home: https://github.com/hallamlab/TreeSAPP
+  license: GNU General Public v3 (GPLv3)
```

Comment:
Since the tarball contains a LICENSE file, you should add: `license_file: LICENSE` here instead of adding it to the recipe folder.